Cargo.toml

```diff
@@ -115,7 +115,7 @@ etcd-client = { git = "https://github.com/MichaelScofield/etcd-client.git", rev
 fst = "0.4.7"
 futures = "0.3"
 futures-util = "0.3"
-greptime-proto = { git = "https://github.com/GreptimeTeam/greptime-proto.git", rev = "2c14c6e22dfe957f40bb88dd01fb8530656de89b" }
+greptime-proto = { git = "https://github.com/Kelvinyu1117/greptime-proto", branch = "refactor/kelvin-add-query-context" }
```

Comment:
Just a remarker for updating the `greptime-proto`

Reply:
I believe you may want to merge the proto file first?

Reply:
We could merge https://github.com/GreptimeTeam/greptime-proto/pull/156 once this PR gets two approvals.

Reply:
@Kelvinyu1117 proto is merged

Reply:
updated proto's repo.